sys/arm64/include/cherireg.h

```diff
@@ -116,8 +121,13 @@
 #define	CHERI_PERMS_KERNEL						\
 	(CHERI_PERM_GLOBAL | CHERI_PERM_LOAD | CHERI_PERM_LOAD_CAP)	\
 
+/*
+ * XXX Will probaby want non-executive PCC when we add comparmentalization
+ * support, but make all code caps executive for now.
+ */
```

Comment:
```suggestion
```
As far as I can see this is only used in early startup so defintiely needs the flag.